pkg/snapshot/snapshotter/garbagecollector.go

```diff
@@ -142,7 +146,10 @@ func (ssr *Snapshotter) RunGarbageCollector(stopCh <-chan struct{}) {
 							ssr.logger.Infof("GC: Deleting chunk for old full snapshot: %s", path.Join(snap.SnapDir, snap.SnapName))
 							if err := ssr.config.store.Delete(*snap); err != nil {
 								ssr.logger.Warnf("GC: Failed to delete snapshot %s: %v", path.Join(snap.SnapDir, snap.SnapName), err)
+								metrics.GCSnapshotCounter.With(prometheus.Labels{metrics.LabelKind: "chunk", metrics.LabelSucceeded: metrics.ValueSucceededFalse}).Inc()
```

Comment:
s/chunk/Chunk/
To keep in line with the naming convention (`Full` and `Incr`).

Would it be better to add `"Chunk"` as a constant named `SnapshotKindChunk`?